tvm/src/pass/ir_mutator.cc

```diff
@@ -541,6 +541,15 @@ Expr IRMutator::Mutate_(const Cast* op, const Expr& e) {
   }
 }
 
+Expr IRMutator::Mutate_(const CastStr* op, const Expr& e) {
+  std::string value = op->value;
+  if (value == op->value) {
```

Comment:
A minor thing. I remember there is an API (like `is_same`) to check if two expressions are the same. Please use that API in case that `==` operator may return wrong results.

Reply:
This is because the value is string, but not Expr, this is why I use == instead of same_as()